webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.jsx',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'bundle.js',
+    publicPath: '/',
+  },
+  devtool: 'inline-source-map',
+  devServer: { contentBase: './dist' },
+  module: {
+    loaders: [
+      {
+        test: /\.jsx?$/,
```

Comment:
Meaning of ```/\.jsx?$/,```?

Reply:
https://github.com/FACN3/Water-you-talking-about/pull/25#discussion_r166539866

in this case it's looking for filenames that end with `.jsx` or `.js` the `?` makes the letter before optional